Sources/Sentry/PrivateSentrySDKOnly.mm

```diff
@@ -1,4 +1,5 @@
 #import "PrivateSentrySDKOnly.h"
+#import "SentryAppStartMeasurement.h"
```

Comment:
When we're doing some changes like this to prepare to another PR, I would suggest to add a little bit more information on the PR's description to explain the motivation, because based on this PR alone I dont know why you're moving headers around.

Reply:
Ups sorry. Yes will do that.